### ISSUE TITLE
Feature Request: Unread messages & mentions

### ISSUE THREAD

**Issue by ShadiestGoat** [rigormorrtiss/discordo#231]
It would be cool to have channels with mentions to be highlighted red, and with unread messages to be bold or italic or something similar.

**Comment by WyntrHeart** [comment]
a number in brackets before the name of the user or channel could show how many unread messages there are. Like this:   [2]UserName

**Comment by kars0n777** [comment]
Anyone working on this? This would greatly improve the program, and the original post is from 2 years ago...

**Comment by anakojm** [comment]
Yes, this would allow me to use the program as I can’t manually check every channel of every server I am in to see if there are new messages.

**Comment by myth0genesis** [comment]
I'm looking for something like this, too. And perhaps the ability to bind a key to jump between servers and channels with unread messages, similar to using Alt + Shift + Arrow Up and Alt + Shift + Arrow Down in the native client.
